alot/db/utils.py

```diff
@@ -298,16 +298,17 @@ def extract_headers(mail, headers=None):
 
 
 def extract_body(mail, types=None, field_key='copiousoutput'):
-    """
-    returns a body text string for given mail.
-    If types is `None`, `text/*` is used:
-    The exact preferred type is specified by the prefer_plaintext config option
-    which defaults to text/html.
+    """Returns a body text string for given mail.
+
+    If types is `None`, `text/*` is used the exact preferred type is specified
```

Comment:
you've dropped the colon after 'used' here..
I must say that I'm also not a fan of the the original docstrings here anyways

Reply:
I just went ahead and rewrote the docstring, so now it actually references the config option it checks and is hopefully a little clearer.

Reply:
much better now, thanks.